support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutFormGifting.jsx

```diff
@@ -351,7 +351,7 @@ function WeeklyCheckoutFormGifting(props: PropTypes) {
             title="Your card details"
           >
             <StripeProviderForCountry
-              country={props.country}
+              country={props.deliveryCountry}
```

Comment:
actually the backend chooses the account based on the currency (AUD vs Other), not the country https://github.com/guardian/support-frontend/blob/5ddfca4d311ad85a3bfb3d32cba37353f0448246/support-config/src/main/scala/com/gu/support/config/StripeConfig.scala#L18